cli/cmd/root.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/edgelesssys/era/era"
+	"github.com/spf13/cobra"
+)
+
+var eraConfig string
+var insecureEra bool
+
+const eraString = `
+Path to remote attestation config file in json format, if none provided the newest configuration will be loaded from github
+`
+
+var rootCmd = &cobra.Command{
+	Use:   "marblerun",
+	Short: "marblerun cli short description",
+	Long:  `marblerun cli long description`,
+}
+
+// Execute starts the CLI
+func Execute() error {
+	return rootCmd.Execute()
+}
+
+func init() {
+
+	rootCmd.AddCommand(newManifestCmd())
+	rootCmd.AddCommand(newNamespaceCmd())
+	rootCmd.AddCommand(newInstallCmd())
+	rootCmd.AddCommand(newRecoverCmd())
+	rootCmd.AddCommand(newRootCACmd())
+}
+
+// VerifyCoordinator verifies the connection using era
+func VerifyCoordinator(host string, configFilename string, insecure bool) (string, error) {
```

Comment:
coordinator verification will be needed for other commands aswell, which is why the function is currently in root.go. Maybe this should be moved to its own file?

Reply:
Makes sense. Yes, we should probably move it to its own file because as I understand it, having root.go is some pattern of Cobra and we should then keep it free of our custom code.